CONTRIBUTING.md

```diff
@@ -337,6 +337,11 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
+Please check with track to which you are submitting an exercise to see if they have any additional guidance on how to order their exercises.
```

Comment:
Hmm. Now next question. Does this sentence seem redundant with the below "Each language track might have additional requirements"? Should I leave it as-is?

Or, I could combine them. "Each language might have additional guidance on how to order their exercises or additional requirements on new exercise files; check the README in the repository for the track"